js/mailalerts.js

```diff
@@ -33,20 +33,36 @@ function  addNotification() {
     success: function (resp) {
       resp = JSON.parse(resp);
 
-      $('div.js-mailalert > span').html('<article class="alert alert-info" role="alert" data-alert="success">'+resp.message+'</article>').show();
+
+      $('div.js-mailalert > span').html('<article class="mt-1 alert alert-' + (resp.error ? 'danger' : 'success') + '" role="alert" data-alert="' + alertProperties.data + '">'+ (resp.error ? 'error' : 'success')+'</article>').show();
```

Comment:
```suggestion
      $('div.js-mailalert > span').html(`<article class="mt-1 alert alert-${resp.error ? 'danger' : 'success'}" role="alert" data-alert="${ alertProperties.data }">${ resp.error ? 'error' : 'success' }</article>`).show();
```

Reply:
Don't do this, we've no transpilers on this project, we should avoid using modern JS for compatibillity

Reply:
May be we should move this to classic theme ?

Reply:
It's the module responsibility, it's the module's code, I don't think so, if we ship this module without this code, it won't work on past classic theme version and it could be at the origin of compatilbity issues

Reply:
Oh sorry. In this case, we need to move code to webpack like here : PrestaShop/blockreassurance#34

Reply:
Yes, that would be a thing, but adding a complex transpiler for only one file is a bit overkill

Maybe we could be able to provide a basic obfuscated configuration like nuxt or other project does to simplify this process

Reply:
Let's keep it simple